apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -193,7 +193,31 @@ async function getFeaturedConsultations(consultation, playlistItems) {
   return featured;
 }
 
-async function processPageConsultation(page) {
+async function processPageConsultationDocument(page, api, context) {
+  const { query } = context;
+
+  const { title, ...rest } = query;
+
```

Comment:
```suggestion
const { blocks } = page;

blocks.push({
        slug: "embedded-document-viewer",
        options: rest,
        title,
      });
      
      return page
```
Why are we set blocks

Reply:
@koechkevin For now we just need one block containing the required document details. We do not need the rest.